app/templates/components/public/session-item.hbs

```diff
@@ -11,7 +11,7 @@
     </div>
     <br>
     <div class="ui grid stackable">
-      {{#if hideImage}}
+      {{#if (and hideImage session.speakers)}}
```

Comment:
check this while computing `hideImage`, because we are in essence trying to hide the image, is session doesn't have speakers. Keep the least amount of logic in template files.